### PR TITLE
docs: added repo build guidelines in README.md

### PR DESCRIPTION
**Pull request by VoltanFr**
Helps developer start discovering the repo for coding.

### CODE REVIEW

**Review by tivie — CHANGES_REQUESTED** [review]
- fix numbering of ordered lists
 - add node (v12) as pre-requisite.

**Comment by tivie** [comment]
thank you!

File: README.md [hunk 1]
```diff
@@ -446,6 +446,20 @@ var showdown    = require('showdown'),
     converter = new showdown.Converter({ extensions: ['myExtension'] });
 ```
 
+## Building
+
+Building your clone of the repository is easy.
+> Prerequesites: [npm](https://www.npmjs.com/package/npm) and [npx](https://www.npmjs.com/package/npx) must be installed.
+
+1. run `npm install`: installs all the needed third-party packages into the [node_modules](/node_modules)  folder, based on the contents of [package-lock.json](/package-lock.json) and [package.json](/package.json).
```
Comment (bandantonio) [inline README.md:454]:
```suggestion
1. run `npm install`.
```
`npm install` doesn't do any magic here, and if you are talking about developers as the target audience for this section, we can safely omit this kinda  excessive description.

File: README.md [hunk 2]
```diff
@@ -446,6 +446,20 @@ var showdown    = require('showdown'),
...
+1. run `npx grunt build`(see [Gruntfile.js](/Gruntfile.js)). This:
```
Comment (bandantonio) [inline README.md:455]:
```suggestion
1. run `npx grunt build` (see [`Gruntfile.js`](/Gruntfile.js)). This command:
```

File: README.md [hunk 3]
```diff
@@ -446,6 +446,20 @@ var showdown    = require('showdown'),
...
+
+    1. Cleans the repo.
+    1. Checks code quality ([JSHint](https://jshint.com/) and [ESLint](https://eslint.org/)).
+    1. Runs tests defined in the repo.
```
Comment (bandantonio) [inline README.md:459]:
```suggestion
    1. Runs tests.
```

File: README.md [hunk 4]
```diff
@@ -446,6 +446,20 @@ var showdown    = require('showdown'),
...
+    1. Creates the [distributable file](/showdown.js) in the [dist](/dist) folder.
+    1. Creates the [minified file](/showdown.min.js) in the same folder.
```
Comment (bandantonio) [inline README.md:461]:
```suggestion
    1. Creates the [distributable](/showdown.js) and [minified](/showdown.min.js) files in the [dist](/dist) folder.
```

**Review by bandantonio — CHANGES_REQUESTED** [review]

**Comment by VoltanFr** [comment]
> * fix numbering of ordered lists

I generally prefer numbering all items as 1., because this eases the maintenance of the list (no need to change all numbers when you insert or delete an item). But if you prefer real numbers, I'll proceed.

File: README.md (hunk 2 above)
Comment (VoltanFr) [reply README.md:455]: Sure you want to put backquotes inside a link caption? Sounds a bit strange to me.
And if we do that, I think we want to do the same for the link to the dist folder, just below?

Comment (bandantonio) [reply README.md:455]: It's just a way to highlight important words, filenames, and commands.
To keep things consistent, it's better to highlight the `dist` folder as well. Thanks for spotting this.

**Review by bandantonio — APPROVED** [review]

**Review by tivie — APPROVED** [review]

**Comment by tivie** [comment]
merged. Thank you @VoltanFr

**Pull request merged**